src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
 
 export const App = () => {
+  const [userID, setValue] = useState(0);
+  const [titles, setTitles] = useState('');
+  const [savedTodos, setTodos] = useState(todosFromServer);
+  const [errorTitle, setNoTitle] = useState(false);
+  const [errorValue, setNoValue] = useState(false);
+
+  const todos: Todo[] = savedTodos.map(todo => ({
+    ...todo,
+    user: getUser(todo.userId),
+  }));
+
+  const lastesID = savedTodos.map(todo => todo.id);
```

Comment:
```suggestion
  const todoIDs = savedTodos.map(todo => todo.id);
```
A bit confusing name